tools/docker/Dockerfile.aliyunlinux3

```diff
@@ -120,32 +120,27 @@ RUN wget http://www.etallen.com/cpuid/cpuid-20200211.x86_64.tar.gz && \
     cp ./cpuid-20200211/cpuid /usr/bin/ && \
     rm -rf /tmp/cpuid-20200211*
 
+
+# Download the Occlum source
+ARG OCCLUM_BRANCH
+WORKDIR /root
+RUN git clone -b $OCCLUM_BRANCH https://github.com/occlum/occlum && \
+    cp -r /root/occlum/tools/toolchains/* /tmp/ && \
+    cp /root/occlum/tools/docker/start_aesm.sh /opt/occlum/
+
 # Install Occlum toolchain
-COPY toolchains/musl-gcc /tmp/musl-gcc
 WORKDIR /tmp
 RUN cd musl-gcc && ./build.sh && ./install_zlib.sh && rm -rf /tmp/musl-gcc
 ENV PATH="/opt/occlum/build/bin:/usr/local/occlum/bin:$PATH"
 
 # Install glibc
-COPY toolchains/glibc /tmp/glibc
 WORKDIR /tmp
 RUN cd glibc && ./build.sh && rm -rf /tmp/glibc
 
 # Install Occlum Golang toolchain
-COPY toolchains/golang /tmp/golang
 WORKDIR /tmp
-RUN yum install -y go && cd golang && ./build.sh && rm -rf /tmp/golang
-ENV PATH="/usr/local/occlum/golang/bin:$PATH"
-
-# Install Occlum Java toolchain (JDK 11)
-ARG JDK11_PATH=/opt/occlum/toolchains/jvm/java-11-openjdk
-COPY --from=alpine /usr/lib/jvm/java-11-openjdk $JDK11_PATH
-RUN rm $JDK11_PATH/lib/security/cacerts
-COPY --from=alpine /etc/ssl/certs/java/cacerts $JDK11_PATH/lib/security/cacerts
-COPY toolchains/java /tmp/java
-WORKDIR /tmp
-RUN cd java && ./install_dragonwell.sh && rm -rf /tmp/java
-ENV PATH="/opt/occlum/toolchains/jvm/bin:$PATH"
+RUN cd golang && ./build.sh && rm -rf /tmp/golang
```

Comment:
Does the golang still work in the docker image?

Reply:
Yes, it is working